app/services/people/retrieve_activities.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module People
+  class RetrieveActivities
+    def self.call(person)
+      activities = NomisClient::Activities.get(person.latest_nomis_booking_id)
+
+      activities.map do |activity_json|
```

Comment:
not sure if the suffix `_json` add any info, it sounds that `activity_json` is a json string.

Maybe we could call it `nomis_activity`, or just `activity`

Reply:
🤷

Reply:
activity makes me think of the model. nomis activity might work better